.github/workflows/secrel4prs.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         uses: ./.github/actions/publish-images
         with:
           target_env: ${{ env.TARGET_ENV }}
-          access_token: ${{ secrets.GITHUB_TOKEN }}
+          access_token: ${{ secrets.ACCESS_TOKEN_GRADLE_BUILD }}
```

Comment:
Tested [here](https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/4000430160) -- I'll continue to tweak this in PR #963, so it's okay if this fails.